lib/cog_api/http/base.ex

```diff
@@ -24,6 +24,13 @@ defmodule CogApi.HTTP.Base do
     end)
   end
 
+  def put(%Endpoint{}=endpoint, resource, params) do
+    rescue_econnrefused(fn ->
+      body = Poison.encode!(params)
+      HTTPotion.put(make_url(endpoint, resource), body: body, headers: make_headers(endpoint))
+    end)
+  end
```

Comment:
We were using `patch` everywhere. I thought we settled on using `put` instead, so I added it. At some point we may want to update all the other calls.